README.md

```diff
@@ -12,7 +12,7 @@ This repository is for providing feedback and documentation on the Pylance langu
 
 # Features
 
-![ screencast ](screencap.gif)
+<img src=images/all-features.gif>
```

Comment:
Why the HTML tag and the the markdown tag as was used previously? The image tag works, but curious about the change.